tests/antithesis/compose.go

```diff
@@ -22,9 +23,56 @@ import (
 
 const bootstrapIndex = 0
 
+// Creates docker-compose configuration for an antithesis test
+// setup. Configuration is via env vars to simplify usage by main entrypoints. If
+// the provided network includes a subnet, the initial DB state for the subnet
+// will be created and written to the target path.
+func GenerateComposeConfig(network *tmpnet.Network, baseImageName string) error {
+	targetPath := os.Getenv("TARGET_PATH")
+	if len(targetPath) == 0 {
+		return errors.New("TARGET_PATH environment variable not set")
+	}
+
+	imageTag := os.Getenv("IMAGE_TAG")
+	if len(imageTag) == 0 {
+		return errors.New("IMAGE_TAG environment variable not set")
+	}
+
+	// Subnet testing requires creating an initial db state for the bootstrap node
+	if len(network.Subnets) > 0 {
+		avalancheGoPath := os.Getenv("AVALANCHEGO_PATH")
+		if len(avalancheGoPath) == 0 {
+			return errors.New("AVALANCHEGO_PATH environment variable not set")
+		}
+
+		pluginDir := os.Getenv("AVALANCHEGO_PLUGIN_DIR")
+		if len(pluginDir) == 0 {
+			return errors.New("AVALANCHEGO_PLUGIN_DIR environment variable not set")
+		}
```

Comment:
Rather than using `errors.New` directly in the `return` - can we define them as variables? There is a longstanding goal I've had to add `goerr113` as a linter... But there have been a few blockers (but this would be prevented by that linter)

Reply:
In this case it isn't super important (more of a consistency style thing). But in other locations it's important to be able to test the different error conditions more naturally (with `errors.Is`)